examples/file-io.rs

```diff
@@ -0,0 +1,69 @@
+//! Run ruperf against some file calls
+//!
+//! This is a small program to run ruperf against. Write and read a file. This 
+//! should call `syscall_write` and `syscall_read`. 
+//! 
+//! Usage:
+//!     ruperf stat ./file-io
+use std::fs;
+use std::io::prelude::*;
+use std::path::PathBuf;
+use std::env;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "File-io", about = "A test program to run ruperf against")]
+struct Opt {
+    /// Number of write to a file
+    #[structopt(short, long, default_value = "100000")]
+    count: usize,
+}
+
+static WRITE_STRING: &str = "The quick brown fox jumped over the lazy dog\n";
+
+fn main() {
+    let opt = Opt::from_args();
+    let writes = opt.count;
+    let tmp_dir = env::temp_dir();
+    let mut path = PathBuf::from(tmp_dir);
+    path.push("ruperf-fileio-test");
+    path.set_extension("txt");
+
+    if path.exists() {
```

Comment:
not sure the best default behavior is to delete the file if it exists. What if someone happens to have an existing file they want to keep? Maybe we should be generating a filename that will be unique and allow the filepath to be specified as an input? For instance you can use the process id or generate a hash to guarantee uniqueness